File: beecrowd/1005 - Média 1.go
package main

import "fmt"

func main() {
	var nota1, nota2 float64

	fmt.Scan(&nota1)
	fmt.Scan(&nota2)

	media := (nota1*3.5 + nota2*7.5) / 11.0

	fmt.Printf("MEDIA = %.5f\n", media)
}

File: beecrowd/1005 - Média 1.kt
import java.util.*

import java.util.Locale

fun main(args: Array<String>) {

	val nota1 = readLine()!!.toDouble()
    val nota2 = readLine()!!.toDouble()

    val media = (3.5 * nota1 + 7.5 * nota2) / 11.0

    println("MEDIA = %.5f".format(Locale.US, media))
}